src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExceptionsPipelineTests.cs

```diff
@@ -285,17 +312,20 @@ public TestExceptionsStore(int instanceThreshold = 1)
                 _instanceThreshold = instanceThreshold;
             }
 
-            public void AddExceptionInstance(IExceptionsNameCache cache, ulong groupId, string message, DateTime timestamp)
+            public void AddExceptionInstance(IExceptionsNameCache cache, ulong groupId, string message, DateTime timestamp, ulong[] stackFrameIds, int threadId)
             {
                 StringBuilder typeBuilder = new();
-                FunctionData throwingMethodData;
+                WebApi.Stacks.FunctionData throwingMethodData;
+                CallStack callStack;
                 try
                 {
                     Assert.True(cache.TryGetExceptionGroup(groupId, out ulong exceptionClassId, out ulong throwingMethodId, out _));
 
-                    NameFormatter.BuildClassName(typeBuilder, cache.NameCache, exceptionClassId);
+                    WebApi.Stacks.NameFormatter.BuildClassName(typeBuilder, cache.NameCache, exceptionClassId);
```

Comment:
Is the namespace qualification necessary here?

Reply:
I believe so, due to the ambiguity between `WebApi.Stacks` and `WebApi.Models` for some things (like `CallStack`).

Reply:
But there's only one `NameFormatter` class [here](https://github.com/dotnet/dotnet-monitor/blob/main/src/Microsoft.Diagnostics.Monitoring.WebApi/Stacks/NameFormatter.cs#L15); the namespace only changes if it is code in the startup hook assembly, which this is not.

Reply:
I just meant that we can't include `WebApi.Stacks` and `WebApi.Models` without needing to qualify something because of the ambiguity around `CallStack` - I can add a `using CallStack = Microsoft.Diagnostics.Monitoring.WebApi.Models.CallStack` to address that though.